src/css/profile/mobile/common/listview.less

```diff
@@ -140,25 +140,34 @@ tau-expandable {
 			}
 		}
 
-		&.ui-li-has-icon {
```

Comment:
Is it really  delete the ui-li-has-icon left margin??
Is it correct?

Reply:
Yes, it is.
Icon already has a margin value.
```
margin: 0 12 * @px_base;
```